pywatts/summaries/mape_summary.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import numpy as np
+
+from pywatts.summaries.metric_base import MetricBase
+
+logger = logging.getLogger(__name__)
+
+
+class MAPE(MetricBase):
+    """
+    Module to calculate the Mean Absolute Percentage Error (MAPE)
+
+    :param offset: Offset, which determines the number of ignored values in the beginning for calculating the MAPE.
+                   Default 0
+    :type offset: int
+    :param filter_method: Filter which should performed on the data before calculating the MAPE.
+    :type filter_method: Callable[[np.ndarray, np.ndarray], Tuple[np.ndarray, np.ndarray]]
+    """
+
+
+    def _apply_metric(self, p, t):
+        non_zero_indices = t.nonzero()[0][t.nonzero()[0] >= self.offset]
+        return np.mean(np.abs((p[non_zero_indices] - t[non_zero_indices]) / t[non_zero_indices]))
```

Comment:
I think you need to multiply the mean with 100. See [Mape wikipedia](https://en.wikipedia.org/wiki/Mean_absolute_percentage_error)

Reply:
In my lecture "Forecasting" I got to know MAPE without the factor 100 (so I guess it is a conventional decision and in the book "Forecasting: Principles and Practice" from Hyndman and Athanasopoulos MAPE is also introduced with factor 100). Hence, I've added the factor 100.